test/seeds/index.ts

```diff
@@ -22,7 +22,7 @@ export const seededAcceptTransactionId = 'd8eb8a94-222b-4481-b315-1dcbf2e07079'
 export const seededDemandAId = 'ae350c28-f696-4e95-8467-d00507dfcc39'
 
 export const seededMatch2Id = 'f960e4a1-6182-4dd3-8ac2-6f3fad995551'
-export const exampleDate = '2023-03-24T10:40:47.317Z'
+export const exampleDate = '2023-01-01T00:00:00.000Z'
```

Comment:
in some cases I like to use `mock-date` module so date kinda can be now

Reply:
yeah and we use `sinon` so I could use `fakeTimers`. This was a smaller change to make it more obvious this was a mock value